src/algebra/lie/basic.lean

```diff
@@ -169,6 +169,8 @@ linear_map.map_add (f : L₁ →ₗ[R] L₂) x y
 
 @[simp] lemma map_lie (f : L₁ →ₗ⁅R⁆ L₂) (x y : L₁) : f ⁅x, y⁆ = ⁅f x, f y⁆ := morphism.map_lie f
 
+@[simp] lemma map_zero (f : L₁ →ₗ⁅R⁆ L₂) : f 0 = 0 := (f : L₁ →ₗ[R] L₂).map_zero
```

Comment:
```suggestion
@[simp] lemma morphism.map_zero (f : L₁ →ₗ⁅R⁆ L₂) : f 0 = 0 := (f : L₁ →ₗ[R] L₂).map_zero
```
to enable dot notation. Frankly, `map_lie` should be renamed too, but then you need to rename the field to `map_lie'` which is more work.

Reply:
Agreed but I contend this is work for a future PR.

I'm close to reaching a minor milestone in Lie theory and once that's done, if this is still pending, I can fold it into plans that I have a refactor.

I'll make a proposal to the community first before I make a PR but it should be something like:

- splitting `lie/basic.lean` into multiple files,
- sorting out some bad naming,
- improving namespacing, 
- mopping up a bunch of details like the `map_zero`, `map_lie` point here.

Reply:
@eric-wieser I've finally addressed this point [here](https://github.com/leanprover-community/mathlib/pull/6179)